test/scenarios/synapse/output/extflatten/src/synapse/azext_synapse/tests/latest/example_steps.py

```diff
@@ -104,7 +104,8 @@ def step_big_data_pool_create(test, checks=None):
              checks=[])
     test.cmd('az synapse big-data-pool wait --created '
              '--name "{myBigDataPool}" '
-             '--resource-group "{rg}"',
+             '--resource-group "{rg}" '
+             '--workspace-name "{myWorkspace}"',
              checks=checks)
```

Comment:
I don't see similar change on help.py. Is it because we are already using show command info to make the wait command help info ?

Reply:
yes, the help.py was using show's  parameter for wait, no need to change